src/components/QueryBuilder.vue

```diff
@@ -30,6 +30,9 @@
 					:disabled="selectedPropertyValueRelation === propertyValueRelation.Regardless"
 				/>
 			</div>
+			<div class="querybuilder__add-condition">
```

Comment:
I think this can just go inside the component. I think it makes more sense there.

Reply:
Done